project_import_export.go

```diff
@@ -46,6 +46,7 @@ type ImportStatus struct {
 	PathWithNamespace string     `json:"path_with_namespace"`
 	CreateAt          *time.Time `json:"create_at"`
 	ImportStatus      string     `json:"import_status"`
+	ImportError       string     `json:"import_error"`
```

Comment:
Should we also add `CorrelationID` and `FailedRelations`? As I notice that are also mentioned in the docs...

Reply:
Yes that could make sense. I just added the minimum to get the reason when the import fails.